PokemonRampUp/Model/Ability.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Ability.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+struct Ability: Decodable {
+    let ability: AbilityName
```

Comment:
Why is the type called AbilityName? 

what is the url property used for?

Reply:
what is the url property used for?
 
answer: Thinking about it now there is actually no reason I need to store the URL information. I removed it. 

Why is the type called AbilityName?

answer: As I removed the url from AbilityName I think it is a better description of what it does

Reply:
You dont need AbilityName

Reply:
I think I fixed the issue